scripts/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu
```

Comment:
@haboy52581 Xiaodong, I just tried to use this Dockerfile locally and found that it took me more than 30 minutes to complete it.

We might need to rely on this Dockerfile in our Onebox deployment, K8s deployment and daily CI workflow. 

Please try a light-weighted option like CIRROS or others.

Reply:
I think it really depends on the scenario you want to use, if this file is used for internally then please let me know why the repo you want to use; if this is for outside world, then this is very fast if you are running on the normal box since that's for dockerhub connection

Reply:
Okay let us discuss various options.